prettier.cjs

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs/promises')
+
+const comment = ' // temp comment to fool Prettier'
+
+const readFileOrig = fs.readFile
+fs.readFile = async (path, ...args) => {
+    if (!path.toString().endsWith('.js')) {
+        return readFileOrig(path, ...args)
+    }
+
+    const code = await readFileOrig(path, ...args)
+    return (
+        code
+            // don't collapse function call if multiline
+            .replaceAll(/ (\w+)\(\n/g, ` $1(\n${comment}\n`)
+            // don't collapse array definition if multiline
+            .replaceAll(/ \[\n/g, ` [\n${comment}\n`)
```

Comment:
I've tried to rewrite files on the fly to prevent array and fn call collapsing, and this seems to be working